core/src/main/java/org/bitcoinj/core/ECKey.java

```diff
@@ -739,12 +760,7 @@ public static boolean verify(byte[] data, ECDSASignature signature, byte[] pub)
      */
     public static boolean verify(byte[] data, byte[] signature, byte[] pub) throws SignatureDecodeException {
         if (Secp256k1Context.isEnabled()) {
-            try {
-                return NativeSecp256k1.verify(data, signature, pub);
-            } catch (NativeSecp256k1Util.AssertFailException e) {
-                log.error("Caught AssertFailException inside secp256k1", e);
-                return false;
-            }
+            return NativeSecp256k1.verify(data, signature, pub);
```

Comment:
Should we also handle exception here?

Reply:
Added try/catch.

I think Vlad has a point.